package.json

```diff
@@ -61,7 +61,9 @@
     "build:css:base": "lessc --strict-math=on --include-path=node_modules src/basestyle/index.less lib/basestyle/index.css",
     "build:css:theme": "lessc --strict-math=on --include-path=node_modules src/default-theme/index.less lib/default-theme/index.css",
     "clean": "rimraf docs && rimraf lib && rimraf test/build && rimraf test/coverage",
+    "clean:all": "npm run clean && cd jupyterlab && npm run clean && cd ..",
     "clean:examples": "node scripts/cleanexamples.js",
+    "reset": "npm run clean:all && rimraf jupyterlab/node_modules && rimraf node_modules && npm install && npm run build:all",
```

Comment:
On second thought, then it's confusing what the difference between `build` and `rebuild` is. I'd prefer to take out the build step and call it `reset` or something else that indicates that it gives you a clean slate.